main.js

```diff
@@ -124,8 +130,36 @@ for (var i = 0 ; i < courses.length ; i++) {
    }
 }
 
+sortNameDown.style.display="none";
+withoutSort.style.display="none";
+sortName.style.display= "inline-block" ;
+
+function toggleSortOrder() {
```

Comment:
we can pass to this function the type of sort (up or down) and based on it we can do the switch statement